libs/blocks/global-navigation/utilities/keyboard/index.js

```diff
@@ -25,6 +25,43 @@ const cycleOnOpenSearch = ({ e, isDesktop }) => {
   }
 };
 
+const openProfile = ({ e, el }) => {
+  const button = e.target.closest(`${selectors.signIn}, ${selectors.profileButton}`);
+  if (button?.getAttribute('aria-expanded') === 'false') {
+    e.target.click();
+    el.querySelector(selectors.profileDropdown)?.focus();
+    return true;
+  }
+  return false;
+};
+
+const getProfileItems = ({ e }) => {
+  const focusableElements = document.querySelectorAll(selectors.profileDropdown);
```

Comment:
Would naming this something like `profileDropdownLinks` or `profileDropdownButtons` make a bit more sense? I got a bit confused initially, expecting for this to just represent a dropdown, not focusable items inside it.